core-rust-bridge/src/main/java/com/radixdlt/protocol/ProtocolConfig.java

```diff
@@ -74,22 +74,19 @@
 import java.util.Map;
 
 public record ProtocolConfig(
-    String genesisProtocolVersion,
     ImmutableList<ProtocolUpdateTrigger> protocolUpdateTriggers,
     Map<String, byte[]> rawProtocolUpdateContentOverrides) {
 
   public static final String GENESIS_PROTOCOL_VERSION_NAME = "babylon-genesis";
+  public static final String ANEMONE_PROTOCOL_VERSION_NAME = "anemone";
+  public static final String BOTTLENOSE_PROTOCOL_VERSION_NAME = "bottlenose";
+
+  public static final String LATEST_PROTOCOL_VERSION_NAME = BOTTLENOSE_PROTOCOL_VERSION_NAME;
```

Comment:
this seems easy to miss when introducing a new update :thinking: wdyt about moving it to `NetworkStatusTest`, as it's only used there?

Reply:
Well my thinking was that this might well be useful in other tests :). Indeed, we use a similar pattern across the Scrypto code base.

I view the `NetworkStatusTest` as ensuring this constant stays up to date. Will add some comments...